docs/legacy/exploring-es-data.asciidoc

```diff
@@ -49,27 +60,20 @@ The following examples enable you to interact with {es}'s REST API.
 One possible way to do this is using {kib}'s
 {kibana-ref}/console-kibana.html[{dev-tools-app} console].
 
-Indices, templates, and index-level operations can also be manged via {kib}'s
+Data streams, templates, and index-level operations can also be manged via {kib}'s
 {kibana-ref}/managing-indices.html[Index management] panel.
 
-To see an overview of existing indices, run:
+To see an overview of existing data streams, run:
 ["source","sh"]
 ----
-GET _cat/indices/apm*
-----
-// CONSOLE
-
-To query all documents collected with a specific APM Server version:
-["source","sh",subs="attributes"]
-----
-GET apm-{version}-*/_search
```

Comment:
I removed this example as data streams aren't defined by the version of the APM Server. Let me know if there's something better to include instead.